iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -624,19 +646,38 @@ TEST_F(PortManager_test, PortDestroy)
             EXPECT_THAT(subscriber1.getSubscriptionState(), Eq(iox::SubscribeState::WAIT_FOR_OFFER));
         }
     }
+}
 
-    // re-create the ports of process p2
-    publisherData2 =
+TEST_F(PortManager_test, CleanupProcessP2ChangesStatesOfPortsInProcessP1)
```

Comment:
After looking at the entire test it seems this is more an integration test than a unit test and I think this needs to be reverted, since the semantic of the test are changed if it just split. In the original test, the `publisher1` is offering the all the time and the `subscriber1` is subscribed all the time. So `subscriber1` will automatically be re-subscribed when `publisher2` offers the service. After the split, this is not the case anymore.

Reply:
okay so no need to split the previous test?

Reply:
no, it's a big one but if it is split, it wouldn't mimic the previous behavior with re-subscription